docs/netdata-agent/configuration/optimizing-metrics-database/change-metrics-storage.md

```diff
@@ -1,180 +1,88 @@
+| Tier |     Resolution      | Time Limit | Size Limit |
+|:----:|:-------------------:|:----------:|:----------:|
+|  0   |  high (per second)  |  14 days   |   1 GiB    |
+|  1   | middle (per minute) |  3 months  |   1 GiB    |
+|  2   |   low (per hour)    |  2 years   |   1 GiB    |
 
-## Calculate the system resources (RAM, disk space) needed to store metrics
+With these defaults, Netdata requires approximately 4 GiB of storage space (including metadata).
 
-### Effect of storage tiers and disk space on retention
+## Retention Settings
 
-3 tiers are enabled by default in Netdata, with the following configuration:
+> **In a parent-child setup**, these settings manage the shared storage space utilized by the Netdata parent agent for storing metrics collected by both the parent and its child nodes.
 
-```
-[db]
-    mode = dbengine
-    
-    # per second data collection
-    update every = 1
-    
-    # number of tiers used (1 to 5, 3 being default)
-    storage tiers = 3
-    
-    # Tier 0, per second data
-    dbengine multihost disk space MB = 256
-    
-    # Tier 1, per minute data
-    dbengine tier 1 multihost disk space MB = 128
-    dbengine tier 1 update every iterations = 60
-    
-    # Tier 2, per hour data
-    dbengine tier 2 multihost disk space MB = 64
-    dbengine tier 2 update every iterations = 60
-```
-
-The default "update every iterations" of 60 means that if a metric is collected per second in Tier 0, then
-we will have a data point every minute in tier 1 and every hour in tier 2. 
-
-Up to 5 tiers are supported. You may add, or remove tiers and/or modify these multipliers, as long as the 
-product of all the "update every iterations" does not exceed 65535 (number of points for each tier0 point).
+You can fine-tune retention for each tier by setting either the time limit or size limit to zero. This enables various
+retention strategies as shown in the table below:
 
-e.g. If you simply add a fourth tier by setting `storage tiers = 4` and define the disk space for the new tier, 
-the product of the "update every iterations" will be 60 \* 60 \* 60 = 216,000, which is > 65535. So you'd need to reduce  
-the `update every iterations` of the tiers, to stay under the limit.
+| Setting                        | Retention Behavior                                                                                                                        |
+|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| Time Limit = 0, Size Limit > 0 | **Time-based only:** data is stored for a specific duration regardless of disk usage.                                                     |
+| Size Limit = 0, Time Limit > 0 | **Space-based only:** data is stored until it reaches a certain amount of disk space, regardless of time.                                 |
```

Comment:
This is the other way around. Time limit = 0, disables time based retention check